src/zframe.c

```diff
@@ -571,9 +571,9 @@ zframe_test (bool verbose)
 
     //  @selftest
     //  Create two PAIR sockets and connect over inproc
-    zsock_t *output = zsock_new_pair ("@inproc://zframe.test");
+    zsock_t *output = zsock_new_pair ("@tcp://127.0.0.1:9001");
```

Comment:
Argh...I stuffed that up, sorry. I was going to use zauth to return some meta data I could test, which doesn't work on inproc as far as I know. I didn't end up doing that, but now that I've reverted it to inproc, I'm getting a segfault... I'll need a few minutes to figure it out. I'm not very experienced with C.